### PR TITLE
[hybris/aosp/LA.UM.9.12.r1] Add module for video hardware codecs, update submodules

### PR DESCRIPTION
**Pull request by Thaodan**

### CODE REVIEW

**Review by sledges — CHANGES_REQUESTED** [review]
Please update kernel [straight from upstream](https://github.com/mer-hybris/android_kernel_sony_msm/compare/hybris-sony-aosp/LA.UM.9.12.r1...sonyxperiadev:aosp/LA.UM.9.12.r1) in a separate PR for cleanliness and ease of reviewing, then when that's merged, rebase this PR.

**Comment by Thaodan** [comment]
> Please update kernel [straight from upstream](https://github.com/mer-hybris/android_kernel_sony_msm/compare/hybris-sony-aosp/LA.UM.9.12.r1...sonyxperiadev:aosp/LA.UM.9.12.r1) in a separate PR for cleanliness and ease of reviewing, then when that's merged, rebase this PR.

Adjusted.

**Comment by sledges** [comment]
https://github.com/mer-hybris/android_kernel_sony_msm/pull/101

File: .gitmodules [hunk 1]
```diff
@@ -10,6 +10,14 @@
 	path = techpack/data-kernel
 	url = https://github.com/sonyxperiadev/kernel-techpack-data-kernel
         branch = aosp/LA.UM.9.12.r1
+[submodule "techpack/display"]
+	path = techpack/display
+	url = https://github.com/sonyxperiadev/kernel-techpack-display-driver
+        branch = aosp/LA.UM.9.12.r1
+[submodule "techpack/video"]
+	path = techpack/video
+        branch = aosp/LA.UM.9.12.r1
```
Comment (sledges) [inline .gitmodules:19]: nitpick: use tab instead of spaces

File: .gitmodules [hunk 2]
```diff
@@ -10,6 +10,14 @@
 	path = techpack/data-kernel
 	url = https://github.com/sonyxperiadev/kernel-techpack-data-kernel
         branch = aosp/LA.UM.9.12.r1
+[submodule "techpack/display"]
+	path = techpack/display
+	url = https://github.com/sonyxperiadev/kernel-techpack-display-driver
```
Comment (sledges) [inline .gitmodules:15]: nitpick: a chance to unify with most of others by appending `.git`

**Pull request merged**